lib/puppet_x/msutter/templates/invoke_dsc_resource.erb

```diff
@@ -0,0 +1,74 @@
+$ErrorActionPreference = "Stop"
+$PuppetModulesFolder = Join-Path $env:ProgramFiles 'WindowsPowerShell\Modules\PuppetVendoredModules'
+if (!(Test-Path($PuppetModulesFolder))) {
+    & cmd.exe /c mklink /d "$PuppetModulesFolder" "<%= vendored_modules_path %>" | Out-Null
+}
+
+function Set-RefreshMode
+{
+  param($mode)
+
+  $configOutputPath = [IO.Path]::GetTempPath()
+
+  [DscLocalConfigurationManager()]
+  Configuration LCMSettings {
+      Node localhost
+      {
+          Settings
+          {
+              RefreshMode = $mode
+          }
+      }
+  }
+
+  LCMSettings -OutputPath $configOutputPath
+
+  Set-DscLocalConfigurationManager -Path $configOutputPath
```

Comment:
What happens when this call fails (i.e. if there is another operation in progress)?